src/schemas/validation/schema.yaml

```
@@ -230,6 +230,52 @@ $defs:
         type: array
         items:
           $ref: '#/$defs/parameter-or-reference'
+      additionalOperations:
+        type: object
+        additionalProperties:
+          $ref: '#/$defs/operation'
+        not:
+          anyOf:
+            - properties:
+                get:
+                  $ref: '#/$defs/operation'
+              required:
+                - get
+            - properties:
+                put:
+                  $ref: '#/$defs/operation'
+              required:
+                - put
+            - properties:
+                post:
+                  $ref: '#/$defs/operation'
+              required:
+                - post
+            - properties:
+                delete:
+                  $ref: '#/$defs/operation'
+              required:
+                - delete
+            - properties:
+                options:
+                  $ref: '#/$defs/operation'
+              required:
+                - options
+            - properties:
+                head:
+                  $ref: '#/$defs/operation'
+              required:
+                - head
+            - properties:
+                patch:
+                  $ref: '#/$defs/operation'
+              required:
+                - patch
+            - properties:
+                trace:
+                  $ref: '#/$defs/operation'
+              required:
+                - trace
```

How about using something like this:
```suggestion
        propertyNames:
          not:
            pattern: '^(get|put|[pP][oO][sS][tT]|delete|options|head|patch|trace)$'
```

Of course adding the lower/Upper pattern for all verbs in addition to `POST`.

(Being lazy I only added it for POST to make the new "fail" testcase actually fail.)

---

resolving this since we have another thread with a potential solution